stdlib/LinearAlgebra/src/structuredbroadcast.jl

```diff
@@ -126,6 +126,7 @@ fails as `zero(::Tuple{Int})` is not defined. However,
 """
 iszerodefined(::Type) = false
 iszerodefined(::Type{<:Number}) = true
+iszerodefined(::Type{<:AbstractArray}) = true
```

Comment:
Isn't this dependent on the `eltype` of the AbstractArray?

Reply:
Yes. Done.